sw/device/sca/otbn_vertical/ecc256_keygen_serial.h

```diff
@@ -50,6 +50,26 @@ void ecc256_en_masks(const uint8_t *enable, size_t enable_len);
  */
 void ecc256_set_seed(const uint8_t *seed, size_t seed_len);
 
+/**
+ * Simple serial 'c' (set constant) command handler.
+ *
+ * The constant must be `kEcc256SeedNumBytes` bytes long.
+ *
+ * @param C Value of the C constant.
+ * @param len Length of the C constant.
+ */
+void ecc256_set_c(const uint8_t *C, size_t len);
+
+/**
+ * Simple serial 'e' (secret keygen fvsr key batch mode) command handler.
+ *
+ * Takes a number of traces that has to be captured in one batch as input.
+ *
+ * @param data Value for trace count.
+ * @param data_len Length of trace count input.
+ */
```

Comment:
From the function names, it's not clear what the difference between
- `ecc256_ecdsa_keygen_fvsr_key_batch()`, and
- `ecc256_ecdsa_secret_keygen_batch()`

are and this comment header doesn't provide more insight. Please rename the two functions and update the comments to provide more insight. As long as the actual command letter, i.e., `e` and `b` remain the same things stay compliant with the host side code.

Reply:
I've renamed `ecc256_secret_keygen_batch()` to `ecc256_ecdsa_keygen_fvsr_seed_batch()` which I think is a better fit with our naming convention. I've also added descriptions for these two functions.